lib/Command/Room/ListCommand.php

```diff
@@ -0,0 +1,111 @@
+<?php
+declare(strict_types=1);
+/**
+ * @copyright Copyright (c) 2020 Daniel Rudolf <nextcloud.com@daniel-rudolf.de>
+ *
+ * @author Daniel Rudolf <nextcloud.com@daniel-rudolf.de>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace OCA\Talk\Command\Room;
+
+use OC\Core\Command\Base;
+use OCA\Talk\Manager;
+use OCA\Talk\Room;
+use OCP\IConfig;
+use Symfony\Component\Console\Helper\Table;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class ListCommand extends Base {
```

Comment:
I'm unsure about this command. I feel it exposes the user's privacy too much/easily. I know that the admin could also check this in the database, but it feels wrong to have it available like this.

Reply:
Since all commands require a room token, I was feeling like there must be a way to retrieve a list of the available room tokens via CLI. However, to be honest, without a list of all participants it can get quite hard to find the right room anyway (since room names don't have to be unique)... On the other hand, it just lists the bare minimum - and no participants. So one could argue that it isn't showing sensitive data anyway. So, as you can see, I'm not sure either :laughing:

Reply:
Well showing the name can be very intrusive. E.g. if management has a conversation: "Can we fire sysadmin1". But then again they could also check it in the database. but it's not so easy, so :/

Reply:
Removed, see 0c57a68